rshim.spec.in

```diff
@@ -15,9 +15,7 @@ Source0: https://github.com/mellanox/rshim-user-space/archive/%{name}-%{version}
 BuildRequires: gcc, autoconf, automake, pkgconfig, make
 BuildRequires: pkgconfig(libpci), pkgconfig(libusb-1.0)
 
-%if (0%{?rhel} >= 8 || 0%{?fedora} > 0) && "0%{?ctyunos}" == "0"
-Requires: kernel-modules-extra
-%endif
+Requires: kmod(cuse.ko)
```

Comment:
I am wondering whether this syntax works in all our existing BFBs.  ++@vladsokolovsky in case he might have some comments.

Reply:
Good comment. Let's hold-off on this change. Let me test it on different distributions we need to support and see whether it works.

Reply:
That'll be great. It's not related to BFB, but need to try different hosts (at least centos 7.x)